### PR TITLE
add js studygroup

### PR DESCRIPTION
**Pull request by mattphoto**
starting on Sundays js studygroup led by Matt Kim

### CODE REVIEW

File: app/views/static_pages/_sg_js.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+ <div class="studygroup-side pull-left" >   <!-- changed #side to .studygroup-side -->
+  <h3>The Rails track!</h3>
```
Comment (eriktrautman) [inline app/views/static_pages/_sg_js.html.erb]: Are you sure this is the Rails track?

**Comment by eriktrautman** [comment]
Where should people go to make this happen?  There aren't any links to Hangouts or calendar events or anything.  If you need a Meetup group to run through, you can use one of ours (http://www.meetup.com/Learn-Web-Development/)

**Comment by eriktrautman** [comment]
The .DS_Store file got pulled in somehow.  Are you using our .gitignore?  It should prevent those from getting pulled in.

File: app/views/static_pages/_sg_js.html.erb (hunk 1 above)
Comment (mattphoto) [inline app/views/static_pages/_sg_js.html.erb]: not sure what happened there. Will be fixed in the next pr

**Comment by eriktrautman** [comment]
You can just update your fork and the pull request will automatically pull in the changes, so no need in the future to kill the old PR.

                                                           
Erik S. Trautman
Founder,  Viking Code School
The premier online program for learning professional web development

**Pull request closed without merging**